backend/src/main/java/com/empire/World.java

```diff
@@ -2064,6 +2047,66 @@ void eatFood() {
 		}
 
 		void growPopulations() {
+			// Emigration
+			class Emigration {
+				final Region source;
+				final Region sink;
```

Comment:
Mostly a semantic argument being made here...source/sink usually means the creation (seemingly out of thin air) and subsequent consumption of some object/quantity.  I think source/destination might be a more accurate description here.  Doesn't really matter though.

Reply:
Done